nix/mobile/jsbundle/default.nix

```diff
@@ -68,8 +68,8 @@ stdenv.mkDerivation {
   buildPhase = ''
     # Assemble CLASSPATH from available clojure dependencies.
     # We append 'src' so it can find the local sources.
-    export CLASS_PATH="$(find ${deps.clojure} \
-      -iname '*.jar' | tr '\n' ':')src"
+    export CLASS_PATH="src:$(find ${deps.clojure} \
```

Comment:
Comment worthy?

Reply:
It's worth, anything we mess here is like playing with fire.